### ISSUE TITLE
Make uses of GitHub labels to trigger the script

### ISSUE THREAD

**Issue by allamand** [jklukas/git-push-fork-to-upstream-branch#7]
Hi,

It would be very nice if it is possible to use a label on the PR in order to trigger a webhook that will execute the script on our behalf.

As an example we can see on the helm/charts repo that they uses a specific label `ok-to-test` which then trigger the pipeline to be executed:

https://github.com/helm/charts/pull/14414#issuecomment-498320987

I'm not sure if we could have a bot like k8s-robot that monitor thoses labels and then execute your script to push the branch into trigger-integration which will restart all tests ?

thanks

**Comment by jklukas** [comment]
Indeed, that would require some service to monitor the repository in question and it would need to have permissions to push to the relevant branch. That's beyond the scope of anything I plan to do with the script here, but it's potentially something you could pursue building.

**Comment by allamand** [comment]
Thanks I'll see what we can do or reuse for that ;)

**Comment by weitzman** [comment]
Great idea. Seems like Github Actions could solve this. For example, here is an [an action that facilitates pushing into another repo](https://github.com/ad-m/github-push-action).

Thanks for this script. Its small and useful!
